src/components/LoginSignUp/AuthenticationService.js

```diff
@@ -13,6 +13,10 @@ import Realm from "realm";
 // eslint-disable-next-line import/extensions
 import realmConfig from "realmModels/index";
 
+import { log } from "../../../react-native-logs.config";
+
+const userLog = log.extend( "user" );
```

Comment:
I find it helpful to use the name of the file or module when adding logging prefixes like this, so how about logger `const = log.extend( "AuthenticationService" )` instead? If there's a smart way to get the name of the file dynamically, even better, but I think it's ok to hardcode it.